charts/templates/cluster-global-sidecar.yaml

```diff
@@ -95,19 +95,29 @@ spec:
       serviceAccountName: global-sidecar
       containers:
         - name: global-sidecar
+          env:
+            - name: PROBE_PORT
+              value: {{ default 8080 $gs.probePort | quote }}
```

Comment:
this default port value is very likely to conflict with business svc port. we should replace it with an uncommon one.

Reply:
change to 18181